config.xml

```diff
@@ -130,11 +79,15 @@
     <allow-intent href="geo:*" />
     <platform name="android">
         <allow-intent href="market:*" />
+        <preference name="CodePushDeploymentKey" value="rVLju2NoxbcCyC9f3h2biGcCRx8Y4JBH-fLXW" />
     </platform>
```

Comment:
I think they should not be. You can have the key but what matters is the login and password info without with you cant deploy to codepush. You can see on similar project here https://github.com/rangle/code-push-demo/blob/master/config.xml , and binary mobile here https://github.com/binary-com/mobile/blob/master/config.xml and many more. To my understanding, i don't what someone can do with the keys , you need authentication to login to codepush before you can deploy and with the authenticatio, you can retrieve these keys themselves. So they are not sensitive.